### ISSUE TITLE
Revise the sitemap diagram based on comments in #51 to make sure the propoposed website content and their hierarchies are comprehensible to the project team

### ISSUE THREAD

**Issue by gissoo** [Princeton-CDH/geniza#72]
Here is the [link to the revised sitemap. ](https://www.figma.com/file/HpGBOZi9lO8B3nCAx3d4Fj/Princeton-Geniza?node-id=425%3A12)

**Note:**
- I have added notes in blue where more context was needed. Please read.
- Please do not pay attention to what the "explore fragments" and "explore words" and "citation/scholarship records" will entail – we will discuss during our meeting. (just view the levels at which they are placed). 

**Description:**
- this is the revised sitemap, covering the content and functionality, and levels of the following pages on the site: 
1. homepage, 
2. cluster search, 
3. browse documents by cluster (the page shown once a cluster is selected for further exploration), 
4. search results (the page shown when there is an input in the search box)
5. document details 
6. Citation/Scholarship Records
7. Contact Us
8. About with its subpages: 1. credits, how to cite, data exports, technical and FAQ

- Pages that may be out of scope depending on data and priorities: (will discuss at our meeting)
1. Citation/Scholarship Records
2. Explore Fragments
3. Explore words 

**Questions:**
1. Does the sitemap make sense to you? Does the legend make sense? How about the page levels?  If not, please say why
2.Do you have any additions to what's considered as "Content" vs. "functionality" on any of the pages? Anything that's missing? Or you consider unnecessary?
3. Would you want to revise the name of any of the pages? For instance, which one makes more sense, "Citation records" or "Scholarship Records"? 
4. Is there anything that you expected to see and is missing?

**Comment by gissoo** [comment]
@mrustow @sluescher Please read the notes above, view the sitemap through the link provided, and answer the questions. 
Please let me know if there is anything that doesn't make sense.

**Comment by sluescher** [comment]
@mrustow commented on Figma, I'll add mine here. Sorry for the delay.
- Does the sitemap make sense to you? Does the legend make sense? How about the page levels? If not, please say why
   - Yes, it makes sense. I have a question regarding the data exports: I thought it'd be attached to the search/document details. how do you envision finding what one wants to download?
- Do you have any additions to what's considered as "Content" vs. "functionality" on any of the pages? Anything that's missing? Or you consider unnecessary?
  - Somewhere in the about, I'd like to see a page exploring the process we went through to get the database/website running, decision we made, and how the fragments ended in Europe and the US and what ethical concerns that may raise.
- Would you want to revise the name of any of the pages? For instance, which one makes more sense, "Citation records" or "Scholarship Records"?
  - I would say scholarship records.
- Is there anything that you expected to see and is missing? 
  - See second response.

**Comment by gissoo** [comment]
> @mrustow commented on Figma, I'll add mine here. Sorry for the delay.
> 
>     * Does the sitemap make sense to you? Does the legend make sense? How about the page levels? If not, please say why
>       
>       * Yes, it makes sense. I have a question regarding the data exports: I thought it'd be attached to the search/document details. how do you envision finding what one wants to download?
>

- @sluescher thanks for writing!! There is an "Export Data" in the "functionality" section of "Browse documents by cluster" and "Search Results" – the page "Data Exports" under "About" is what I'm proposing if we want to make large amounts of data downloadable grouped by for instance doc type or themes ...it would be good to discuss this on Thursday as part of the sitemap doc – this is something that was done for S&Co [here](https://shakespeareandco.princeton.edu/about/data/) but not sure how much it would make sense for Geniza. 

- Also, do we need a "Export Data" on the individual doc detail page? It's currently not included in the sitemap. 

>     * Do you have any additions to what's considered as "Content" vs. "functionality" on any of the pages? Anything that's missing? Or you consider unnecessary?
 
>       * Somewhere in the about, I'd like to see a page exploring the process we went through to get the database/website running, decision we made, and how the fragments ended in Europe and the US and what ethical concerns that may raise.
> 

- This is interesting. Parts of that would be covered in "Technical" – are you envisioning a page that's more about "Data"? Or a page that would group all of the "Data" and "Technical"/software content? What would you call the page you're talking about? Are you envisioning a detailed kind of process? Or highlights/what is important?

>     * Would you want to revise the name of any of the pages? For instance, which one makes more sense, "Citation records" or "Scholarship Records"?      

>       * I would say scholarship records.
>

- Great! 

>     * Is there anything that you expected to see and is missing?
>       
>       * See second response.

**Comment by gissoo** [comment]
Marina's comments on Figma: 
- Re: Sort by Date under "Cluster Search" Marina mentions: "Date of dated or datable docs, right? In addition to this it would be good to have by date of when it was added to the Db." @mrustow do you mean users would see when a certain document was added to the Db? – Could you say more on why it would matter for the user? I'm curious 

- Excited about the "Has Scholarship Records" in the "Cluster Search" page 

- Re: "Cluster Search" page Marina mentions: "Sorry if this comment is missing context or forgetting a decision — but it would be great to have a non-cluster search on the landing page, either here or the level above here, so the hard-core users can do targeted searches with as few clicks as possible." – @mrustow the idea is that there won't be any "cluster search" (we might need to rethink this label) – the idea is that the search box that will exist is only going to give users the list of documents as search results – clusters are only what will appear below the search box and by interacting with them users can navigate through sub clusters and documents – does this make sense to you?

- Re: "FAQ" under "About" Marina mentions: "I would want to see this called out in an earlier level. It could also be good to distinguish between a basic "how-to" guide and a FAQ. I feel like a lot of websites lately have been dumping basic information into FAQs, whereas I think of FAQs as second-level questions, once you've looked around. So I'd want a how-to earlier, and possibly also the FAQ visible earlier." – @mrustow, great, what are you envisioning the "how-to" page be for? Why would it exist and what would it entail?  – I'm proposing a "how to cite" page for instance as you might have seen – would this "how-to" page also include the content from the "how to cite" page?

**Comment by richmanrachel** [comment]
Hi @gissoo! Several thoughts/questions:
1) I see that you'll talk about this in the meeting, but just to put my questions in writing: does the "Explore Fragments" function pull up random fragments or curated fragments? And is the "Explore Words" like a word cloud?
2) Thank you for clarifying for Marina that "cluster search" is no longer the accurate term. It would be helpful for you to update the diagram, because I just spent 15 minutes writing and deleting questions about this. I like the idea of a single search where clusters appear in the results. 
3) As for the data export conversation - it does make sense to me that having the full metadata download should be on a separate page in addition to being able to download datasets from search results. As for downloading from individual documents, this could be useful as a way to build your own dataset. From a user's perspective, I wonder if it would be easier to be able to "add to temporary workspace" and then download the whole set before one closes the browser, or if it should be more of a "copy metadata" function that can be pasted into one own's spreadsheet. I have no idea from a developer standpoint if these ideas are feasible. @rlskoeser or @thatbudakguy - are either of these ideas possible?
4) I think that "has translation" and "has citation" will be feasible at level 1  if we get the footnotes working properly. 
5) I think what Marina means in distinguishing between "FAQ" and "How-to" is the difference between conceptual information and practical information. So FAQ would have the "why does this project exist" and "who works on it," whereas the How-To page describes how to best utilize the website's functionality (how to download datasets, etc). How to cite could definitely be folded in here.
6) I'm not sure I understand your distinction well enough between "content" and "function" to help you quite yet. For example, I understand in the Document Details section that all of those functions are actual things that you can actively play with. But why in Search Results is "associated shelfmark" a function rather than content?
7) I agree with Stephanie and Marina that overall things make sense and that "Scholarship Records" is better than "Citations."
Thanks for your hard work!

**Comment by gissoo** [comment]
> Hi @gissoo! Several thoughts/questions:
> 
>     1. I see that you'll talk about this in the meeting, but just to put my questions in writing: does the "Explore Fragments" function pull up random fragments or curated fragments? And is the "Explore Words" like a word cloud?
> 

- thanks for writing @richmanrachel!! "Explore Fragments" and "Explore words" are not included in this issue as mentioned in the description, sorry, I think we need to deprioritize conversing on them as I have mainly focused on the doc detail view and the cluster search navigation and interfaces – maybe we can talk about them when I'm back the next time so that by then I have gathered a bit more information on both of them and from one more researcher in the field – when we do converse about them I'll first focus on the goals for having these on the site and if they are at all applicable at this point, the visual aspects are going to be discussed at even a later stage, won't focus on what it would look like yet. 

>     2. Thank you for clarifying for Marina that "cluster search" is no longer the accurate term. It would be helpful for you to update the diagram, because I just spent 15 minutes writing and deleting questions about this. I like the idea of a single search where clusters appear in the results.
> 

- @richmanrachel I think we need to come up with a better term as I was reading the comments yesterday and will be part of what we will discuss at our meeting tomorrow – When reviewing issues, I think it would definitely save you more time if you can read the comments first and then look at the designs. The workflow is that the designs are revised when everyone has commented and shared feedback on it and when all the questions I have asked have been answered. (in this case I was waiting till we discuss some of these questions tomorrow / or in general when Marina responds to my questions on github, otherwise revising them early has proven to be very inefficient)

>     3. As for the data export conversation - it does make sense to me that having the full metadata download should be on a separate page in addition to being able to download datasets from search results. As for downloading from individual documents, this could be useful as a way to build your own dataset. From a user's perspective, I wonder if it would be easier to be able to "add to temporary workspace" and then download the whole set before one closes the browser, or if it should be more of a "copy metadata" function that can be pasted into one own's spreadsheet. I have no idea from a developer standpoint if these ideas are feasible. @rlskoeser or @thatbudakguy - are either of these ideas possible?
> 

- oh interesting! @richmanrachel What do you mean by "add to temporary workspace"? Could you say more?

>     4. I think that "has translation" and "has citation" will be feasible at level 1  if we get the footnotes working properly.
> 

- great!! 

>     5. I think what Marina means in distinguishing between "FAQ" and "How-to" is the difference between conceptual information and practical information. So FAQ would have the "why does this project exist" and "who works on it," whereas the How-To page describes how to best utilize the website's functionality (how to download datasets, etc). How to cite could definitely be folded in here.
> 

- This makes sense to me – If we take this route I anticipate both "FAQ"s and "How-to"s can become contextually available throughout the site (or at least a link to them).

>     6. I'm not sure I understand your distinction well enough between "content" and "function" to help you quite yet. For example, I understand in the Document Details section that all of those functions are actual things that you can actively play with. But why in Search Results is "associated shelfmark" a function rather than content?
> 

- @richmanrachel we can discuss this further at our meeting, "Associated shelfmark checkmark" / or as I just revised it for now to be "Other documents in the same shelfmark checkmark" should be considered as content, thanks for catching! We were considering them to be more fuzzy but RSK just convinced me now and I lean more towards them being content. 

>     7. I agree with Stephanie and Marina that overall things make sense and that "Scholarship Records" is better than "Citations."
>

- great!! 

>        Thanks for your hard work!
>

Thanks for reviewing and for the feedback!!

**Comment by richmanrachel** [comment]
@gissoo - thanks for the clarifications and workflow overview! 

The "temporary workspace" might be a way to add various documents into a list that's remembered via cookies or something, so that as a user finds documents of interest, they can collate them together in a list that could then be downloaded. It might be too complicated to construct, but it would be a nice feature for folks looking to put together documents that don't necessarily come up in a search together.

**Comment by gissoo** [comment]
Thank you for your feedback here, I will close this issue as we will focus on discussing the search, and the "potential out-of-scope" sections including the "scholarship records" during our future meetings through the sitemap doc. 

Some decisions we made: 

- Rename "Cluster Search" to "Search"
- Have "How-to" page in addition to the "FAQ"s – the content on each of these pages will be need to be decided. (Currently, we are thinking that "How-to" will be more extensive/high-level view type of content and FAQs will be more technical/specific to functionalities)
- Add "Date entered" to both sort and filter under "Search" – to let users know when a document was added into the database
- Be open/anticipate of incorporating sorting by date and filtering options (geographical locations, doc type, and doc dates) for clusters 
- Learn from researchers about their needs for a temporary workspace/collecting documents on the level of individual documents
- will ask about the "data export" page under "About" in a future meeting.
